CBinding/Gui/CTextEditorExtension.cs

```diff
@@ -550,7 +551,7 @@ protected void Run ()
 		[CommandUpdateHandler (MonoDevelop.DesignerSupport.Commands.SwitchBetweenRelatedFiles)]
 		protected void Update (CommandInfo info)
 		{
-			var cp = (CProject)DocumentContext.Project;
+			var cp = (CMakeProject)(DocumentContext.Project as SolutionItem);
```

Comment:
Again, this should be `var cp = (CMakeProject)Owner`. There are a few more but I won't call the rest out individually.